core/primitives/digest.hpp

```diff
@@ -100,16 +103,20 @@ namespace kagome::primitives {
                      Pause,            // 4: N delay
                      Resume>;          // 5: N delay
 
-  struct DecodedConsensusMessage {
+  using UnsupportedDigest_POL1 = Empty;
```

Comment:
As we touched this, maybe also makes sense to introduce unsupported digest "BEEF"?

Reply:
It will be resolved in #1190